lib/rendition.js

```diff
@@ -60,6 +155,26 @@ class Rendition {
         return obj;
     }
 
+    inline() {
+        return Number.isInteger(this.instructions.inlineLimit)
```

Comment:
So, no parsing if `this.instructions.inlineLimit` is a string?

Reply:
Yes, it has to be a number in the request json. No need to allow strings IMO, just increases the complexity.